### PR TITLE
Reproduce pyserini: BM25 Baselines for MS MARCO Passage/Document Ranking Experiment 

### PR DESCRIPTION
**Pull request by mzzchy**
Get exactly same result as provided. Screenshots are post
p_Document
Pyserini_BM25_Passage_2

Environment:
OS: macOS Big Sur 11.4
Python: 3.9.5
Java: openjdk 11.0.5 2019-10-15

Issue encountered:
mvn doesn't set ANSERINI_CLASSPATH properly when compile code
issue

Solved: Manually adding `export ANSERINI_CLASSPATH='/Users/zizeng/Code/anserini/target'` (Change `zizeng` to your username) to `.bash_profile`(or `.zshrc` depends on our shell)

### CODE REVIEW

**Comment by MXueguang** [comment]
did you move the fatjar file into pyserini/pyserini/resources/jars?
following this https://github.com/castorini/pyserini/#development-installation

**Comment by mzzchy** [comment]
Oh, I think I just use the command from `anserini` to set up tools so I didn't notice that part.  I am sorry about that.

**Review by MXueguang — APPROVED** [review]

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -144,3 +144,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results reproduced by [@mayankanand007](https://github.com/mayankanand007) on 2021-05-04 (commit [`a9d6f66`](https://github.com/castorini/pyserini/commit/a9d6f66234b5dd2859a0dc116ef3e38a52d0f81d))
 + Results reproduced by [@rootofallevii](https://github.com/rootofallevii) on 2021-05-14 (commit [`e764797`](https://github.com/castorini/pyserini/commit/e764797081eebf487fa7e1fa34872a59ff97fdf7))
 + Results reproduced by [@jpark621](https://github.com/jpark621) on 2021-06-13 (commit [`f614111`](https://github.com/castorini/pyserini/commit/f614111f014b7490f75e585e610f64f769164dd2))
++ Results reproduced by [@mzzchy](https://github.com/mzzchy) on 2021-07-05 (commit [`dc730ff`](https://github.com/castorini/pyserini/commit/dc730ffb24e1ff2a34999166ac4d14e41b8312d5))
```
Comment (MXueguang) [inline docs/experiments-msmarco-doc.md:147]: seems this commit id does not belong to this repo? 
>This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

**Review by MXueguang — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-doc.md (hunk 1 above)
Comment (mzzchy) [reply docs/experiments-msmarco-doc.md:147]: That URL actually redirects me to my commit to this pr.
Here is how I create the pr.

Add a record to md file with a random commit id.
Change the commit id to the one generated in step 1.
Fix up/Squash the commit.
I don't think there is a way to get the commit id before I make a commit. Or is there a better way to do this

Comment (MXueguang) [reply docs/experiments-msmarco-doc.md:147]: the commit id here should be the last commit of the master branch in this repo, from where you did the replication.

**Review by MXueguang — APPROVED** [review]
looks good to me

**Comment by MXueguang** [comment]
@mzzchy thanks for the replication, I'll leave this PR for @lintool to merge.

**Pull request merged**